### DISCUSSION TITLE
请教选字上屏的问题

### DISCUSSION THREAD

**Discussion by Lantaio** [rime/home#1418] [Q&A] [answered]
我按官方的示例在`key_bindings`文件中指定了几个符号按键为候选项上屏按键，是用的`send: 2`、`send: 3`这种选择第2、第3个候选项的写法。但是在使用输入大写数字功能时，因为数字不是直接上屏，而是在候选框中，这时按那几个选字上屏的符号按键时就不是上屏相应的候选项，而是变成继续输入2、3这些数字了。请问在`key_bindings`文件中除了`send: 2`、`send: 3`这种写法之外，有没有其它更好的明确表示选择第2、第3候选项上屏的写法？

**Comment by iDvel** [comment, marked as answer]
`send` 改成 `send_sequence` 试试：
```yaml
send_sequence: "{Down}{space}"
send_sequence: "{Down}{Down}{space}"
```

**Comment by Lantaio** [comment]
多谢 @iDvel 大神的热心解答！前段时间从你的雾凇拼音输入方案中学到很多Rime设置知识，甚至比在官方wiki学到的还要多，多谢你详尽的注释说明！
谢谢你提供的方法，让我又学到了一个魔法技能。不过我初步思考过和测试过，这种方式还是存在一点小问题。比如说如果我先按了`down`键，再按候选按键的话，上屏的字词就偏移了。当然这个问题不大，我不先按`down`键就行了， 不过如果是调出选项菜单修改输入法设置的时候，按第2个选项展开设置之后，发现当前候选会停留在第2项上，这时就容易搞错选项了。还有就是如果设置候选列表大小为4而实际的候选项只有3个时，如果按上屏第4个候选项那个按键，则会上屏第3个（也就是最后1个）候选项。而正常情况下应该是没有任何反应才对。所以这些微小的差别也有点影响输入准确性。当然除了选项菜单那里无可避免之外，其它都是正常情况下不会发生的。基本上满足了我的需求，谢谢大神。
